presto-mongodb/src/main/java/com/facebook/presto/mongodb/MongoPageSource.java

```diff
@@ -177,11 +178,22 @@ else if (javaType == Block.class) {
         }
     }
 
+    private String toVarcharValue(Object value)
+    {
+        if (value instanceof Collection<?>) {
+            return "[" + String.join(", ", ((Collection<?>) value).stream().map(this::toVarcharValue).collect(toList())) + "]";
```

Comment:
I wonder if we don't break some existing users use cases with this change... but I guess it wasn't very useful before.

@miniway ?

Reply:
Actually, if the value is not a collection of maps, the generated result would be exactly the same as the old way (as done by java.lang.AbstractCollection.toString()).